app.py

```diff
@@ -33,18 +36,17 @@ def celsius():
     return ""
 
 
-@app.route('/weather/<city>/<state>')
```

Comment:
did you intentionally deprecate this route? I think city, state in the URL is preferred over showing the ip address.

Reply:
> did you intentionally deprecate this route? I think city, state in the URL is preferred over showing the ip address.

The route isn't deprecated, I moved it over to the React application. But the Route on react side includes the IP address in the route since that is what you call the API with. Maybe I could remove the IP from the react route, and have the component fetch the coordinates again for those cases where you jump to the route directly (example- http://localhost:5000/Bangalore/Karnataka). In this case, the location coordinates aren't known, and the getLocationByIP API needs to be called again! Let me know if you want me to do this. The current route looks like http://localhost:5000/Bangalore/Karnataka/<coordinates>

Reply:
Oh, and it does not look like an IP address, it looks like your location co-ordinates, so maybe we should be renaming that variable :)

Reply:
Yes, that would be helpful so that route doesn't break